Fix links and list both API specs in README

mkungla opened the pull request.

cvan commented on README.md:
```diff
@@ -2,30 +2,32 @@
 
 ## Communication
 
 * [W3C Community Group: `webvr`](http://www.w3.org/community/webvr/)
```
we ought to update this (cc @toji @NellWaliczek)

cvan replied:
can remove the ": `webvr`" part to be consistent with the [usage above](https://github.com/immersive-web/webxr/pull/309/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20)

cvan commented on README.md:
```diff
@@ -2,30 +2,32 @@
 
-* [WebVR API Specification](https://w3c.github.io/webvr/): main specification for JavaScript API for accessing VR displays.
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/): main specification for JavaScript API for accessing VR and AR devices, including sensors and head-mounted displays.
+* [Legacy WebVR API Specification](https://immersive-web.github.io/webvr/) Legacy WebVR API 1.1 specification for JavaScript API for accessing VR displays. Development of the WebVR API has halted in favor of being replaced the WebXR Device API. Several browsers will continue to support this version of the API in the meantime.
```
`Legacy` -> `: Legacy`

cvan commented on README.md:
```diff
@@ -2,30 +2,32 @@
 ## Specifications
 
-* [WebVR API Specification](https://w3c.github.io/webvr/): main specification for JavaScript API for accessing VR displays.
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/): main specification for JavaScript API for accessing VR and AR devices, including sensors and head-mounted displays.
```
`:  main` -> `: Main`

cvan commented on README.md:
```diff
@@ -2,30 +2,32 @@
-* [WebVR API Specification](https://w3c.github.io/webvr/): main specification for JavaScript API for accessing VR displays.
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/): main specification for JavaScript API for accessing VR and AR devices, including sensors and head-mounted displays.
+* [Legacy WebVR API Specification](https://immersive-web.github.io/webvr/) Legacy WebVR API 1.1 specification for JavaScript API for accessing VR displays. Development of the WebVR API has halted in favor of being replaced the WebXR Device API. Several browsers will continue to support this version of the API in the meantime.
 * [Gamepad API Specification](https://w3c.github.io/gamepad/), introduces a low-level JS API interface for accessing gamepad devices.
```
`, introduces` -> `: Introduces`

cvan commented on README.md:
```diff
@@ -2,30 +2,32 @@
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/): main specification for JavaScript API for accessing VR and AR devices, including sensors and head-mounted displays.
+* [Legacy WebVR API Specification](https://immersive-web.github.io/webvr/) Legacy WebVR API 1.1 specification for JavaScript API for accessing VR displays. Development of the WebVR API has halted in favor of being replaced the WebXR Device API. Several browsers will continue to support this version of the API in the meantime.
 * [Gamepad API Specification](https://w3c.github.io/gamepad/), introduces a low-level JS API interface for accessing gamepad devices.
 * [Gamepad Extensions API Specification](https://w3c.github.io/gamepad/extensions.html): extends the Gamepad API to enable access to more advanced device capabilities.
```
`: extends` -> `: Extends`

mkungla commented:
Updated first commit as requested, Having second look on it I found some inconsistency with links so I added another commit using reference-style link definitions.

toji commented:
Thank you! This has been a messy move, so the help (and simply the extra pair of eyes) is appreciated!
